src/overlays/actors/ovl_En_Ru1/z_en_ru1.c

```diff
@@ -1346,15 +1346,15 @@ void EnRu1_UpdateHeadRotation(EnRu1* this) {
     }
 }
 
-void func_80AEDAE0(EnRu1* this, PlayState* play) {
+void EnRu1_ResetBgCheckFlags(EnRu1* this, PlayState* play) {
     DynaPolyActor* dynaPolyActor = DynaPoly_GetActor(&play->colCtx, this->actor.floorBgId);
 
     if (dynaPolyActor == NULL || dynaPolyActor->actor.id == ACTOR_EN_BOX) {
         this->actor.bgCheckFlags &= ~(BGCHECKFLAG_GROUND | BGCHECKFLAG_WALL | BGCHECKFLAG_CEILING);
     }
```

Comment:
I would like to understand this a bit more to give it a more appropriate name. Its quite odd what its doing, clearing all wall/floor/ceiling detections *specifically* if she is over a non-dynapoly actor (the scene/a void), or above a chest.
Feel like we we need more context of the actions/functions this gets used in to know.

Reply:
I see. I would like to further understand the point of this before we name it. Why should those flags be reset in only those conditions?
Just calling it `ResetBgCheckFlags` makes it sounds like its being reset unconditionally